react/features/conference/components/native/Conference.js

```diff
@@ -17,6 +17,7 @@ import { TestConnectionInfo } from '../../../base/testing';
 import { createDesiredLocalTracks } from '../../../base/tracks';
 import { ConferenceNotification } from '../../../calendar-sync';
 import { Chat } from '../../../chat';
+import { DisplayNameLabel } from '../../../display-name/components/native';
```

Comment:
hmm I'm wondering why the import doesn't happen from `../../../display-name` ? That's what usually is done with importing other features

Reply:
Importing from there works fine, just flow doesn't like it. This is beause of the separatio of ```/web``` and ```/mobile``` folders. If I import from ```/display-name``` then flow argues that there is no such import on the web path - which is true.

Reply:
No more using the workaround of adding an empty file for the platform which has not implemented the component?

Reply:
Well, both are workarounds, so i picked this. I'm not in favour of empty files. But if the team is more in favour of that, i can change it